sr_unix/ydbinstall.sh

```diff
@@ -157,6 +157,33 @@ mktmpdir()
     echo $tmpdirname
 }
 
+# This function is invoked whenever we detect an option that requires a value (e.g. --utf8) that is not
+# immediately followed by a =. In that case, the next parameter in the command line ($2) is the value.
+# We check if this parameter starts with a "--" as well and if so it denotes a different option and not a value.
+#
+# Input
+# -----
+# $1 for this function is the # of parameters remaining to be processed in command line
+# $2 for this function is the next parameter in the command line immediately after the current option (which has a -- prefix).
+#
+# Output
+# ------
+# returns 0 in case $2 is non-null and does not start with a "--"
+# returns 1 otherwise.
+#
+isvaluevalid()
```

Comment:
Do all the platforms we support have "getopt"? Do we want to consider replacing some of this parsing with utilities like that?

Reply:
Good point. All platforms we support will have getopt. Yes it would be nice to change this script to use getopt for command line parsing. But that is a big change and I don't want to get in there right now when we have a working solution. Will keep that in mind for future script development.